### PR TITLE
Hardening/using namespace in headers for style guide

### PR DESCRIPTION
**Pull request by kzangeli**
See title

### CODE REVIEW

File: doc/manuals/contribution_guidelines.md [hunk 1]
```diff
@@ -395,6 +395,16 @@ xyzInit(user,   pwd  host);
...
 
+#### M12 (`using namespace` in header files)
+
+*Rule*: the construct ```using namespace XXX``` must not be used in header files.
+
+*Rationale*: a header file should not decide for the source files that include the header file.
+If a header file contains this construct then all source files that include the header file,
+directly or indirectly are forced to have that construct as well.
+
+*How to check*: easily added to the internal script ```scripts/style_check.sh```, using an intelligent ```grep```.
```
Comment (fgalan) [inline doc/manuals/contribution_guidelines.md]: What about just:

"using an intelligent grep```

?

Comment (kzangeli) [inline doc/manuals/contribution_guidelines.md]: Well, the style_check script is mentioned in other 'how to checks'

Comment (kzangeli) [inline doc/manuals/contribution_guidelines.md]: Fixed in dac25cc

**Comment by fgalan** [comment]
LGTM

**Pull request merged**